src/modules/QtQuick/MouseArea.js

```diff
@@ -9,13 +9,15 @@ QmlWeb.registerQmlType({
     hoverEnabled: "bool",
     mouseX: "real",
     mouseY: "real",
-    pressed: "bool",
+    isPressed: "bool",
     containsMouse: "bool",
     pressedButtons: { type: "variant", initialValue: 0 },
     cursorShape: "enum" // Qt.ArrowCursor
   },
   signals: {
     clicked: [{ type: "variant", name: "mouse" }],
+    pressed: [{ type: "variant", name: "mouse" }],
```

Comment:
Ah, to have an `onPressed` signal…

Hm, looks like there is a confict currently for elements that have properties and signals under the same name.
Will look into it.